plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -23,3 +23,13 @@ if [ -f /usr/libexec/pk-command-not-found ]; then
         return $retval
     }
 fi
+
+# OSX command-not-found support
+# https://github.com/Homebrew/homebrew-command-not-found
+if [ -e /usr/local/bin/brew ]; then
```

Comment:
FWIW: it's not encouraged, but it's possible to install Homebrew in locations other than `/usr/local`.